app/assets/stylesheets/oregon_digital/_home-page.scss

```diff
@@ -71,12 +71,20 @@ td.toggle .input-group-btn:last-child > input[type=submit]:not(:last-child) {
 #collections,
 #items {
   padding-bottom: 5em;
-}
+}:where
```

Comment:
What does this do?

Reply:
Heh, I missed that. I think it autocompleted from me trying to type `:w` to write a save.